picard/tagger.py

```diff
@@ -573,6 +576,9 @@ def handle_command_lookup(self, argstring):
             log.error("Invalid LOOKUP command argument: '%s'", argstring)
 
     def handle_command_lookup_cd(self, argstring):
+        if not _discid:
+            log.error("The Lookup CD function is not available.")
```

Comment:
Let's change the wording here. We have the same check already in https://github.com/metabrainz/picard/blob/master/picard/ui/mainwindow.py#L745 , I think it would be a good idea to use the same wording.

But also different to the check in mainwindow let's keep this here an error (as you currently did it). Trying to execute this command without discid being avaible really counts as an error message.

Reply:
Done, but in a case like this, should I be adding "CDROM: discid library not found - Lookup CD functionality disabled" as a string variable/constant in `picard/disc/__init__.py` and including it in `picard/tagger.py` and `picard/mainwindow.py` to avoid maintining the same message in two different locations?

Reply:
I actually decided to add the string constant to `picard/util/cdrom.py` since that's the module that was already providing `discid` that was used for testing.  If you don't want this done, please let me know and I can revert the changes.